package/yast2-network.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Thu May 25 09:27:09 UTC 2017 - mfilka@suse.com
+
+- bnc#1039532
+  - do not write hostname into /etc/hosts duplicitly for one IP
+  - configure static IPs to resolve to system wide hostname during
+    installation
```

Comment:
no, before the hostname was assigned to every static ip - even that which was configured separately.